src/codegen/templates/property_getter_template.py.mako

```diff
@@ -0,0 +1,165 @@
+<%def name="script_property_getter(attribute)">\
+<%
+        from codegen.utilities.text_wrappers import wrap, docstring_wrap
+    %>\
+    @property
+    def ${attribute.name}(self):
+        """
+        ${attribute.description | docstring_wrap(8, 12)}
+        """
+## Script instantiation of numpy arrays for input parameters that are lists, and ctypes variables for
+## output parameters that will be passed by reference.
+<%
+        instantiation_line = None
+        if not attribute.has_explicit_read_buffer_size:
+            instantiation_line = 'val = {0}()'.format(attribute.ctypes_data_type)
+    %>\
+    %if instantiation_line:
+        ${instantiation_line}
+
+    %endif
+\
+    %if len(attribute.c_function_name) < 33:
+        cfunc = lib_importer.${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}.DAQmxGet${attribute.c_function_name}
+    %else:
+        cfunc = (lib_importer.${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}.
+                 DAQmxGet${attribute.c_function_name})
+    %endif
+\
+## Create argument ctypes types list.
+    %if attribute.calling_convention == 'StdCall':
```

Comment:
There doesn't appear to be an else. So maybe just strip all the logic around checking the `calling_convention` and just assert that `calling_convention == 'StdCall'` in the attribute class?

Reply:
The DAQmx C API should only use cdecl for varargs functions, but it looks like a few other functions use it, possibly by mistake. I don't think any attribute-specific functions use it, which is probably why there was no else case in this template.